scripts/seed.js

```diff
@@ -247,6 +247,12 @@ async function createPermissions() {
         scope: 'global',
         description: 'Add anyone to any circle in the system, no matter if the circle is joinable or not but still respecting that bound circles can only hold members of the same body. This also allows to add yourself to any circle and thus can be used for a privilege escalation'
     });
+    permissions.manageAntennaCritera = await Permission.create({
```

Comment:
```suggestion
    permissions.manageAntennaCriteria = await Permission.create({
```

Also needs to be fixed in the usages